can/io/sql.py

```diff
@@ -0,0 +1,127 @@
+from can.listener import BufferedReader
+from can.message import Message
+
+import sys
+import time
+import threading
+import sqlite3
+import logging
+
+log = logging.getLogger('can.io.sql')
```

Comment:
Just in case someone would create a writer for MySQL or PostreSQL or whatever, maybe this module should be named sqlite?